typespec-tests/tsp/enum.tsp

```diff
@@ -60,6 +60,7 @@ enum OperationStateValues {
   Failed,
 }
 
+@usage(Usage.input)
```

Comment:
I change it to `input`, to have the setter generated (so that I can see the `unitValue` now settable.